sections/semantics-embedded-content.include

```diff
@@ -3623,12 +3623,63 @@
   For images that have captions, markup generators should use the
   <{figure}> and <{figcaption}> elements to provide the image's caption.
 
-  As a last resort, implementors should either set the <code>alt</code> attribute to the empty string, under
-  the assumption that the image is a purely decorative image that
-  doesn't add any information but is still specific to the surrounding
-  content, or omit the <code>alt</code> attribute
-  altogether, under the assumption that the image is a key part of the
-  content.
+  As a last resort, implementors should set the <code>alt</code> attribute to the empty
+  string, under the assumption that the image is a purely decorative image that doesn't add any
+  information but is still specific to the surrounding content, or omit the <code>alt</code>
+  attribute altogether, under the assumption that the image is a key part of the content.
+
+  <div class="note">
+    Setting an <{image}> element's <code>alt</code> attribute to the empty string will convey
+    to assistive technologies, such as screen readers, that the image in question has no relative
+    or informative value to the section of the document in which it is found. Being that it serves
+    no informative purpose, a screen reader may choose to completely ignore the presence of the
+    image, in favor of not announcing a graphic which has no alternative text to describe it.
+    In the context of a decorative image, this poses no detriment to the end user, as fully
+    understanding the document is not reliant on parsing the image.
+
+    However, in instances where an image is vital to understanding the contents of a document,
+    an image with an omitted <code>alt</code> attribute is preferable to an <code>alt</code>
+    attribute set to the empty string. In the former example, assistive technology would at
```

Comment:
Minor nit, but suggest changing "would" to "will", to be consistent with the tense of the rest of the content.